index.ar.md

```diff
@@ -0,0 +1,2856 @@
+---
+title: رسالة مفتوحة لإزالة ريتشارد ماثيو ستالمان من جميع المناصب القيادية
+description: نحن الموقعون أدناه ، ندعو مجلس إدارة FSF بأكمله إلى التنحي وإقالة ريتشارد ماثيو ستالمان من جميع المناصب القيادية. لن نستمر في معاناته من سلوكه ، أو إعطائه دور قيادي ، أو عدم قبوله بأيديولوجيته المؤذية والخطيرة بأي شكل آخر.
+
+image: /assets/og-image.png
+twitter:
+  card: summary_large_image
+share_text: I support this call for the entire Board of the FSF to step down and for Richard M. Stallman to be removed from all leadership positions.
+---
+
+{% include share-links.html %}
+
+_2021-03-23_
```

Comment:
```suggestion
_2021-03-23_
```

The image/share metadata will still be in English so we can remove this. You can keep the title/description though if you like.